e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @scroll
+
+describe('Scroll', () => {
+    let testTeam;
+    let testChannel;
+    let otherUser;
+
+    beforeEach(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiCreateUser().then(({user: user2}) => {
+                otherUser = user2;
+
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+                });
+            });
+
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-T2368 Fixed Width setting should not scroll pop and display posts properly', () => {
+        // Creating some post to verify scroll pop and Posts views
+        cy.postMessage('This is the first post');
+        Cypress._.times(18, (postIndex) => {
+            cy.postMessage(`p-${postIndex + 1}`);
+        });
+        cy.postMessage('This is the last post');
+
+        // # Switch the account settings for the test user to enable Fixed width center
+        cy.get('#headerInfo > .style--none').click();
+        cy.get('#accountSettings > .style--none > .MenuItem__primary-text').click();
+        cy.get('#accountSettingsModalLabel > span').should('contain', 'Account Settings');
+        cy.get('#displayButton', {timeout: 500000}).click();
+        cy.get('#channel_display_modeEdit > span').click();
+        cy.get('#settingTitle > span').should('contain', 'Channel Display');
```

Comment:
Using the ID of the element should be good enough. You can ignore the `> span` and make it simpler.

Reply:
:heavy_check_mark: